ludwig/models/llm.py

```diff
@@ -293,7 +304,8 @@ def forward(
         # Cast to float32 for metric computation incase we're using deespeed with
         # reduced precision such as bfloat16.
         for prediction_key, prediction_tensor in outputs.items():
-            outputs[prediction_key] = prediction_tensor.type(torch.float32)
+            if prediction_key != PREDICTIONS:
```

Comment:
So the key already exists in the dictionary, I'm just skipping casting it to float32 since the predictions are tokens and they should be int64 (which is already the case)

Reply:
Oh weird, I thought I had added it in a local commit, guess it somehow didn't get pushed up, will add it back

Reply:
Oh yeah, I added it below the if